api/organisations/permissions/permissions.py

```diff
@@ -88,6 +88,17 @@ class OrganisationPermission(BasePermission):
     def has_permission(self, request, view):
         if view.action == "create" and settings.RESTRICT_ORG_CREATE_TO_SUPERUSERS:
             return request.user.is_superuser
+
+        organisation_id = view.kwargs.get("pk")
+        if organisation_id and not organisation_id.isnumeric():
+            raise APIException("Invalid organisation ID")
```

Comment:
This assumes the 500 status code. Why not use `[ValidationError](https://www.django-rest-framework.org/api-guide/exceptions/#validationerror)` instead?

Reply:
Whoops yep good catch, I wanted 400.

Reply:
Done.